.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```

Comment:
The `lint` npm script would come here too, wouldn't it?

Reply:
changed